### PR TITLE
add documentation

### PR DESCRIPTION
**Pull request by sgny**
Here is what I pushed to the repo (and undid later).

I suppose once @tatchi's bindings are merged, we can pull documentation, etc. from this.

### CODE REVIEW

**Comment by sgny** [comment]
I merged @tatchi's version as he had already contributed the important part. This revised PR adds documentation and brings this repo in line with the template.

However, some remaining issues are
- resolving `@reason-react-native/react-native` vs `reason-react-native`
- whether we refer to the module as `ReactNativeCheckBox` or `ReactNativeCheckbox`
- use `open ReactNative` or not

In the interim, I commited a bug fix for `tintColors`.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,119 @@
-# checkbox
-ReasonML / BuckleScript bindings for @react-native-community/checkbox
+---
```
Comment (MoOx) [inline README.md]:
```suggestion

```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,119 @@
...
+
```
Comment (MoOx) [inline README.md]:
```suggestion
```

**Review by MoOx — CHANGES_REQUESTED** [review]
Looks good! Just a tiny thing to remove :D

**Comment by MoOx** [comment]
@sgny To answer you

- Use `reason-react-native` 
- I would go with "Checkbox" as it's the common spelling (even wikipedia know this)
- `open ReactNative` if that's required, it's not an issue as we are doing this everywhere.

**Comment by sgny** [comment]
Will implement those

One issue, the component is addressed from JS as `CheckBox`. That's why I figured if we should have it spelled like that.

File: README.md (hunk 1 above)
Comment (sgny) [reply README.md]: implemented

**Comment by sgny** [comment]
@MoOx merging this, the only open issue is the name for the module, which is fine as it is.

**Pull request merged**